config-templates/redhat/odf-remote/4.7/custom-parameters.json

```diff
@@ -1,49 +1,49 @@
 [
   {
-    "description": "Specify a name for your ODF storage cluster.",
-    "displayname": "Cluster Name",
+    "description": "A unique name for your deployment. Note that Kubernetes resources can't contain capital letters or special characters. Enter a name that uses only lowercase letters, numbers, hyphens or periods.",
+    "displayname": "Deployment name",
     "name": "ocs-cluster-name",
     "required": "true"
   },
   {
-    "description": "The MON storage size (in Gibibytes). The default value is '20Gi'.",
-    "displayname": "Mon storage size",
+    "description": "The total amount of storage available to the monitoring daemonset. The monitoring deamonset consists of 3 monitoring pods that each mount a disk equal to the 'mon-size'. Monitoring data is replicated across these 3 disks. The default size is '20Gi'.",
+    "displayname": "Monitoring daemonset disk size",
     "name": "mon-size",
     "default": "20Gi",
     "place-holder": "20Gi",
     "regex": "^[0-9]+Gi$",
     "required": "true"
   },
   {
-    "description": "The MON storage class name.",
-    "displayname": "Mon storage class",
+    "description": "The storage class to use when dynamically provisioning devices for the monitor pods.",
+    "displayname": "Monitoring pod storage class",
     "name": "mon-storage-class",
     "required": "true"
   },
   {
-    "description": "The OSD storage size (in Gibibytes). The default value is '100Gi'.",
-    "displayname": "osd storage size",
+    "description": "The amount of storage available to applications in your cluster. The total storage available to your applications is equal to the storage daemonset disk size (osd-size) size multiplied by the number of disks (num-of-osd). The default value is '100Gi'.",
+    "displayname": "Storage daemonset disk size",
     "name": "osd-size",
     "default": "100Gi",
     "place-holder": "100Gi",
     "regex": "^[0-9]+Gi$",
     "required": "true"
   },
   {
-    "description": "The OSD storage class name.",
-    "displayname": "osd Storage Class",
+    "description": "The storage class to use when dynamically provisioning devices for the storage daemonset pods.",
+    "displayname": "Storage Class",
     "name": "osd-storage-class",
     "required": "true"
   },
   {
-    "description": "The number of OSDs. The default value is '1'.",
+    "description": "The number storage of daemonsets that you want to deploy. Each daemonset consists of 3 pods that each mount one disk equal to the osd-size that you specify. For example if you specify 'num-of-osd=2', 2 daemonsets are deployed that each contain 3 pods with one disk per pod for a total of 6 pods and 6 disks. App data is replicated across each daemonset. The default value is '1'.",
```

Comment:
The number of OSD disks you want to provision. The total storage available to your applications is equal to the num-of-osd multiplied by osd-size. App data is replicated across each zone (Replication factor: 3). The default value is '1'.